en/links/index.html

```diff
@@ -27,7 +27,7 @@
 		_paq.push(['setTrackerUrl', u+'piwik.php']);
 		_paq.push(['setSiteId', '1']);
 		var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
-		g.type='text/javascript'; g.async=true; g.defer=true; g.src=u+'piwik.js'; s.parentNode.insertBefore(g,s);
+		g.type='text/javascript'; g.async=true; g.defer=true; g.src='https://cdn.bootcss.com/piwik/3.5.0-rc1/piwik.js'; s.parentNode.insertBefore(g,s);
```

Comment:
matomo 3.5.0 is released recently, we won't want to change the href everytime there is an update...